src/FlaUInspect/Properties/AssemblyInfo.cs

```diff
@@ -24,4 +24,4 @@
 
 [assembly: AssemblyVersion("1.3.0")]
 [assembly: AssemblyFileVersion("1.3.0")]
-[assembly: AssemblyInformationalVersion("1.3.0")]
+[assembly: AssemblyInformationalVersion("1.3.0_Milestone")]
```

Comment:
I guess, version changes are done when releasing, so there is no need to touch these values when implementing a feature or fixing a bug.